vllm/lora/utils.py

```diff
@@ -105,3 +109,45 @@ def parse_fine_tuned_lora_name(name: str) -> Tuple[str, bool]:
             return ".".join(parts[2:-1]), parts[-1] == "lora_embedding_A"
 
     raise ValueError(f"{name} is unsupported LoRA weight")
+
+
+def get_lora_absolute_path(lora_path: str) -> str:
```

Comment:
```suggestion
def get_adapter_absolute_path(lora_path: str) -> str:
```
to make it more generic than just LoRA

Reply:
i will also update the method name in unit test later